### PR TITLE
[Stats Refresh] Add Insights > Posting Activity

### LINKED ISSUE #10591: [Stats Refresh] Add Insights > Posting Activity
Add Insights > Posting Activity card.

Design:
posting_activity

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #10591 

This adds the `Posting Activity` card to the Insights table. It's made up of these components:
- `PostingActivityDay` - Shows the square representing a day, color coded per the post count.
- `PostingActivityMonth` - Shows the month grid, built with `PostingActivityDay`s.
  - A bit of new magic - most months span 5 weeks, however there are a few that span 6. December 2018 is an example. There is now a 6th column in the month grid that will show/hide depending on if it's used or not.
- `PostingActivityLegend` - Shows the `Fewer Posts ..... More Posts` view.
- `PostingActivityCell` - Shows the Posting Activity in the Insights table, which includes:
  - The last three months.
  - The `PostingActivityLegend`.
  - `View more`, which displays a temporary alert.

To test:
- Go to Stats > Insights.
- Verify Posting Activity appears.
- Revel in the beauty.
posting_activity

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott - 

posting_activity

**Comment by SylvesterWilmott** [comment]
Looks great @ScoutHarris , thanks!

**Comment by ScoutHarris** [comment]
@jklausa - I just noticed I forgot to add separator lines on the top and bottom of the Posting Activity cell. Since I've already branched twice from this, I'll just go ahead and add them in the branch I'm in now.

File: WordPress/Classes/ViewRelated/Stats/Insights/Posting Activity/PostingActivityLegend.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    static func colorForCount(_ count: Int) -> UIColor? {
+        switch count {
+        case 0:
+            return Style.PostingActivityRangeColors.lightGrey
+        case 1...2:
+            return Style.PostingActivityRangeColors.lightBlue
+        case 3...5:
+            return Style.PostingActivityRangeColors.mediumBlue
+        case 6...7:
```
Comment (jklausa) [inline WordPress/Classes/ViewRelated/Stats/Insights/Posting Activity/PostingActivityLegend.swift]: damn that's super clever!!!!

**Comment by ScoutHarris** [comment]
Thanks @jklausa !

**Pull request merged**